scripts/text_reading/beautify_cosmos_with_ftfy.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import ftfy
+import json
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument("cosmos_dir", help="path to cosmos json file to prettify")
+parser.add_argument("output_dir")
+args = parser.parse_args()
+input_dir = args.cosmos_dir
+output_dir = args.output_dir
+
+
+for input_file in os.listdir(input_dir):
+    print(input_file)
+    if input_file.endswith("--COSMOS-data.json"):
+        print("using file")
+
+        f = open(os.path.join(input_dir, input_file))
```

Comment:
@maxaalexeeva the way this code is written you are leaving a file pointer open till the Python script exits. If something bad happens during the Python script execution this could actually corrupt the file to where it is no longer usable. Please change this and the line below it to one of the two following options:
```python
loaded_json = None
with open(os.path.join(input_dir, input_file)) as infile:
    loaded_json = json.load(infile)
```
... or ...
```python
loaded_json = json.load(open(os.path.join(input_dir, input_file)))
```

Either one of those options prevents the file pointer from hanging around after loading from JSON. You could also do `f.close()` after line 20 but I always forget that option because I never use it. :-)